packages/framework-provider-aws/src/library/events-adapter.ts

```diff
@@ -110,3 +108,57 @@ export async function storeEvents(
   await dynamoDB.batchWrite(params).promise()
   logger.debug('[EventsAdapter#storeEvents] EventEnvelope stored')
 }
+
+export async function destroyEntity(
+  dynamoDB: DynamoDB.DocumentClient,
+  config: BoosterConfig,
+  logger: Logger,
+  entityTypeName: string,
+  entityID: UUID
+): Promise<void> {
+  logger.debug('[EventsAdapter#destroyEntity] Destroying entity')
+  // BatchWrite limitation, in the future we should move this value to BoosterConfig. We are assuming that this process will be
+  // fit into lambda memory and time execution limits.
+  const MAX_EVENTS_TO_DELETE = 25
+  let nextKey: DocumentClient.Key | undefined = undefined
```

Comment:
You can do this:
```suggestion
  let nextKey: DocumentClient.Key
```

Reply:
Not really since is the type of the LastEvaluatedKey in the scan result.

Reply:
Oh I see, but in that case, you don't have to initialize to `undefined` I believe

Reply:
It was removed @NickSeagull